### PR TITLE
odf opeator test

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-mgmt-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-mgmt-p1-f7

**Comment by LiangquanLi930** [comment]
@liangxia Could you help to approve this PR? thanks

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/hypershift/cucushift-installer-rehearse-aws-ipi-ovn-hypershift-workflow.yaml [hunk 1]
```diff
@@ -4,9 +4,11 @@ workflow:
     cluster_profile: aws-qe
     pre:
     - chain: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-provision
+    - ref: operatorhub-subscribe-odf-operator
     test:
     - ref: cucushift-installer-check
     post:
+    - ref: wait
```
Comment (liangxia) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/hypershift/cucushift-installer-rehearse-aws-ipi-ovn-hypershift-workflow.yaml:11]: Debug code in CI ?

**Pull request closed without merging**